access-control-app/src/cmr/access_control/system.clj

```diff
@@ -31,26 +31,52 @@
   {:default nil
    :parser cfg/maybe-long})
 
+(defconfig log-level
+  "App logging level"
+  {:default "info"})
+
 (defconfig access-control-public-protocol
-  "The protocol to use in documentation examples for the access-control application."
+  "The protocol to use in documentation examples for the access-control application.
+
+  Note: this configuration value is used as-is in local, dev environments
+  and is overridden with ENV variables in remote deployments. In both cases,
+  this configuration information is utilized and required.
+
+  The name given after `defconfig` is important and is used when checking for
+  an ENV variable. The ENV variable for this configuration will be:
+  * CMR_ACCESS_CONTROL_PUBLIC_PROTOCOL"
   {:default "http"})
 
 (defconfig access-control-public-host
-  "The host name to use in links returned by the access-control application."
+  "The host name to use in links returned by the access-control application.
+
+  Note: this configuration value is used as-is in local, dev environments
+  and is overridden with ENV variables in remote deployments. In both cases,
+  this configuration information is utilized and required.
+
+  The name given after `defconfig` is important and is used when checking for
+  an ENV variable. The ENV variable for this configuration will be:
+  * CMR_ACCESS_CONTROL_PUBLIC_HOST"
   {:default "localhost"})
 
 (defconfig access-control-public-port
-  "The port to use in links returned by the access-control application."
+  "The port to use in links returned by the access-control application.
+
+  Note: this configuration value is used as-is in local, dev environments
+  and is overridden with ENV variables in remote deployments. In both cases,
+  this configuration information is utilized and required.
+
+  The name given after `defconfig` is important and is used when checking for
+  an ENV variable. The ENV variable for this configuration will be:
+  * CMR_ACCESS_CONTROL_PUBLIC_PORT"
   {:default 3011
    :type Long})
 
-(defconfig log-level
-  "App logging level"
-  {:default "info"})
-
-(defn public-conf
-  "Public access-control configuration used for generating example requests in documentation"
-  []
+(def public-conf
```

Comment:
I think this has to be a function in order for the configuration to be overridable at runtime.

Otherwise you are going to get the value used at the time of building the jars (which does not have the runtime configuration variables set since we use the same jars for every environment and just set the configuration for runtime).